structopt-derive/src/lib.rs

```diff
@@ -25,7 +26,7 @@ use syn::token::Comma;
 use syn::*;
 
 /// Generates the `StructOpt` impl.
-#[proc_macro_derive(StructOpt, attributes(structopt))]
+#[proc_macro_derive(StructOpt, attributes(structopt, casing))]
```

Comment:
Thus, we can't add that.

Reply:
As mentioned above. I am going to remove it.